tests/test_user_retire.py

```diff
@@ -0,0 +1,64 @@
+"""
+Tests for the user_retire sinks.
+"""
+from unittest.mock import patch
+
+import responses
+from responses.registries import OrderedRegistry
+
+from event_sink_clickhouse.sinks.user_retire import UserRetirementSink
+from event_sink_clickhouse.tasks import dump_data_to_clickhouse
+from test_utils.helpers import FakeUser
+
+
+@responses.activate(  # pylint: disable=unexpected-keyword-arg,no-value-for-parameter
+    registry=OrderedRegistry
+)
+@patch("event_sink_clickhouse.sinks.user_retire.UserRetirementSink.serialize_item")
+@patch("event_sink_clickhouse.sinks.user_retire.UserRetirementSink.is_enabled")
+@patch("event_sink_clickhouse.sinks.user_retire.UserRetirementSink.get_model")
+def test_retire_user(mock_user_model, mock_is_enabled, mock_serialize_item):
```

Comment:
Can we get one more test that exercises the `many` version of this, just to make sure no one breaks it in the future?

Reply:
Sure thing, see https://github.com/openedx/openedx-event-sink-clickhouse/pull/63/files#diff-69e4bc0dbdddfc867532a028e3b4b6764834376c04b2ac62b5ea5e61ee928145R77